src/grammars.js

```diff
@@ -57,7 +55,6 @@ export default class Grammars {
 
       return requests
     } catch (error) {
-      console.log(`Unable to fetch resources due to: ${error}`)
```

Comment:
May be it is worth to save this error to console?

Reply:
ok, will put it back.